templates/firefox-ios.xliff

```diff
@@ -1319,6 +1344,102 @@
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInBackground" xml:space="preserve">
+        <source>Open Link in Background</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab while staying on the currently selected tab. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInNewTab" xml:space="preserve">
+        <source>Open Link in New Tab</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab and switching to that tab at the same time. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.SavePageAs" xml:space="preserve">
+        <source>Save Page As...</source>
+        <note>A label indicating the keyboard shortcut of saving the current web page in a format of the user's choice. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Bookmark" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with Bookmarks. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Edit" xml:space="preserve">
+        <source>Edit</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do within a web page. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.File" xml:space="preserve">
+        <source>File</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take on, and within, a Tab. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.History" xml:space="preserve">
+        <source>History</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with History. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Tools" xml:space="preserve">
+        <source>Tools</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with locally saved items. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```

Comment:
Same comment as above
i'm having a hard time understanding how "Tools" is related to the comment